src/libs/thorvg/tvgSwRenderer.cpp

```diff
@@ -446,7 +446,11 @@ bool SwRenderer::target(pixel_t* data, uint32_t stride, uint32_t w, uint32_t h,
 
 bool SwRenderer::preRender()
 {
+#if LV_USE_VG_LITE_THORVG
```

Comment:
Why is it not defined? `lv_conf_internal.h` is included at the top of the file.

Reply:
It's due to the issue mentioned above. To simulate VG-Lite, this function should be bypassed.

> At present, there is still a problem that thorvg will automatically clear the buffer. The issue is here: https://github.com/thorvg/thorvg/issues/1901.

Reply:
I mean I see this warning:
![image](https://github.com/lvgl/lvgl/assets/7599318/df1428e4-e092-4ba2-918b-39e5579ec68d)

The CI passes, so it should be good, I just wonder why we have this warning.

Reply:
Ah, get it. It should be fixed now. Let wait for CI.